third_party/abseil-cpp/absl/debugging/symbolize.cc

```diff
@@ -28,16 +28,16 @@
 #define ABSL_INTERNAL_HAVE_SYMBOLIZE_WASM
 #endif
 
-#if defined(ABSL_INTERNAL_HAVE_ELF_SYMBOLIZE)
-#include "absl/debugging/symbolize_elf.inc"
-#elif defined(ABSL_INTERNAL_HAVE_SYMBOLIZE_WIN32)
-// The Windows Symbolizer only works if PDB files containing the debug info
-// are available to the program at runtime.
-#include "absl/debugging/symbolize_win32.inc"
-#elif defined(__APPLE__)
-#include "absl/debugging/symbolize_darwin.inc"
-#elif defined(ABSL_INTERNAL_HAVE_SYMBOLIZE_WASM)
-#include "absl/debugging/symbolize_emscripten.inc"
-#else
+// #if defined(ABSL_INTERNAL_HAVE_ELF_SYMBOLIZE)
```

Comment:
@jellefoks obviously I don't want to submit this as is, but xb1 can't use `symbolize_win32.inc`. Do you have any idea of if Starboard should always use `symbolize_unimplemented.inc` or if it should be starboardized?

Reply:
Let's make it unimplemented for now and file a folloup task to check if we ever use this part of absl.

It may well be that we (currently) never rely on absl's symbolization.